Gemfile

```diff
@@ -28,3 +28,6 @@ end
 # Use a local Gemfile to include development dependencies that might not be
 # relevant for the project or for other contributors, e.g.: `gem 'pry-debug'`.
 eval_gemfile 'Gemfile-local' if File.exist? 'Gemfile-local'
+
+# Needed for Rails 7.0
```

Comment:
nit: can you also specify something like `and Solidus < x.x` so we can remove it once support for that version fades away?

Reply:
Since we still support Rails 7.0 even in current `main` I can't know in which version we will potentially remove the support.